lib/osf-components/addon/components/registries/overview-form-renderer/component.ts

```diff
@@ -55,7 +68,26 @@ export default class RegistrationFormViewSchemaBlocks extends Component {
         }
     }
 
+    @task
+    @waitFor
+    async loadCurrentModerator() {
+        try {
+            this.currentModerator = await this.store.findRecord('moderator', this.currentUser.currentUserId!,
```

Comment:
Is this our idiomatic way of determining if the current user is a moderator these days?